src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -345,8 +346,9 @@ ApiGlobe.prototype.createSceneGlobe = function (coordCarto, viewerDiv) {
     this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
 
     var map = new Globe(ellipsoid, gLDebug);
+    var updater = new GlobeUpdater(ellipsoid);
 
-    this.scene.add(map);
+    this.scene.setMap(map, updater);
```

Comment:
In my opinion:
- Either you treat map differently for specific purpose (snap-to-ground for instance) and you need to differentiate the two functions
- Either globe is just another layer, and you remove the map attribute